### PR TITLE
fix: add refreshed token for authorization

### PR DESCRIPTION
**Pull request by dangelovska**

### CODE REVIEW

File: rockwallet/rockwallet/src/Platform/AuthenticationClient.swift [hunk 1]
```diff
@@ -240,7 +240,7 @@ enum HTTPMethod: String {
 
 extension URLRequest {
     mutating func authorize(withToken token: String) {
```
Comment (kenanmamedoff) [inline rockwallet/rockwallet/src/Platform/AuthenticationClient.swift:242]: then you don't need the param?

File: rockwallet/rockwallet/src/Platform/AuthenticationClient.swift [hunk 2]
```diff
@@ -240,7 +240,7 @@ enum HTTPMethod: String {
...
-        setValue(token, forHTTPHeaderField: "Authorization")
+        setValue(UserDefaults.sessionToken, forHTTPHeaderField: "Authorization")
```
Comment (kenanmamedoff) [inline rockwallet/rockwallet/src/Platform/AuthenticationClient.swift:243]: what was wrong exactly?

Comment (dangelovska) [reply rockwallet/rockwallet/src/Platform/AuthenticationClient.swift:243]: it doesn't refresh the session token after user logs out
Screenshot 2024-03-19 at 12 14 09

**Review by kenanmamedoff — APPROVED** [review]

**Comment by kenanmamedoff** [comment]
https://bayes.atlassian.net/browse/EE-716

**Pull request merged**